packages/core/src/sass/app/_common/base/common.scss

```diff
@@ -139,6 +139,12 @@ html,
     &--prominent {
         color: var(--text-prominent);
     }
+    &--tnc {
+        text-transform: lowercase;
```

Comment:
why not make the text in lower case instead of adding this style?

Reply:
resolved 😃